contracts/CToken.sol

```diff
@@ -1134,7 +1134,7 @@ contract CToken is CTokenInterface, Exponential, TokenErrorReporter {
       */
     function _acceptAdmin() external returns (uint) {
         // Check caller is pendingAdmin and pendingAdmin ≠ address(0)
-        if (msg.sender != pendingAdmin || msg.sender == address(0)) {
+        if (msg.sender != pendingAdmin || pendingAdmin == address(0)) {
```

Comment:
FWIW, based on the first clause, `msg.sender` here equals `pendingAdmin`. This (weird) clause was to improve formal verification since (technically speaking) the `msg.sender` *could* be zero, which would violate the spec that pending admin must be set, first. Though, it's not a realistic concern.

Reply:
I see.  Yes, my attempted change introduces obvious problems.  My misunderstanding came about because I can't find the point at which the `admin` value is assigned.